docs/installation.rst

```diff
@@ -63,6 +65,16 @@ versions for dependencies. ::
     (mssenv) $ msui
 
 
+.. hint::
+   For installing MSS on the ARM-MAC Apple devices because not all dependencies
+   are crosscompiled you need to set an additional subdir. ::
```

Comment:
This is not required for develop (at least in CI we don't have to set this). Is is it required for stable? What dependency is not available on ARM?

Reply:
because we use a more modern pyqt version in develop